metricbeat/docs/fields.asciidoc

```diff
@@ -10391,7 +10391,7 @@ consumer offset into partition being read
 *`kafka.consumergroup.meta`*::
 +
 --
-type: text
+type: keyword
```

Comment:
I think this needs to be `text`. This value comes from the `server.name` setting in `kibana.yml`. The setting is meant for display purposes, according to it's documentation:

```
# The Kibana server's name.  This is used for display purposes.
#server.name: "your-hostname"
```
 
So it could be a string like "Marketing's Fantastic Analytics UI". In that case I can see the benefit of letting ES analyze the string. Thoughts?